frameworks/Rust/actix/benchmark_config.json

```diff
@@ -19,6 +19,44 @@
       "notes": "",
       "versus": ""
     },
+    "raw": {
+      "fortune_url": "/fortune",
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "Postgres",
+      "framework": "actix",
+      "language": "Rust",
+      "orm": "Raw",
+      "platform": "None",
+      "webserver": "actix-web",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "actix-raw",
```

Comment:
From my experience these values are useless, as this is what is computed automatically.
What I would suggest is to use a friendly name here, like `Actix, Raw`.

Reply:
I wish TE would recommend to use more friendly names in `display_name`, instead of just cloning the handles that are displayed in the chart.